src/views/Events/index.jsx

```diff
@@ -141,34 +167,175 @@ const Events = () => {
     return (
       <Grid container justifyContent="center" spacing={6}>
         <Grid item xs={12} lg={10} xl={8}>
-          <Card style={{ padding: '0 3vw' }}>
-            <CardContent>
-              <Grid container direction="row" alignItems="center">
-                <Grid item xs={4} />
-                <Grid item xs={4} align="center">
-                  <CardHeader title={searchPageTitle} />
+          <Card style={{ padding: '0 0vw' }}>
+            <CardHeader
+              title={searchPageTitle}
+              style={{
+                backgroundColor: gordonColors.primary.blue,
+                color: gordonColors.neutral.grayShades[50],
+              }}
+            />
+            <Card style={{ padding: '0 3vw' }}>
+              <CardContent>
+                <Grid container direction="row" alignItems="center">
+                  <Grid item xs={4} />
+                  <Grid item xs={4} align="center"></Grid>
+                  <Grid item xs={4} align="right"></Grid>
+                </Grid>
```

Comment:
This whole `Grid` container is defunct now, since it only held the heading before, right?

Reply:
If the cards stay separate, I think that'd be the case. I'll keep this in mind